driver/normalizer/strconv.go

```diff
@@ -1,18 +1,44 @@
 package normalizer
 
 import (
+	"fmt"
 	"strconv"
+	"strings"
 	"unicode/utf8"
 )
 
 // Functions below are copied from strconv.Unquote and strconv.Quote.
 // Original functions are unable to escape/unescape values containing
 // multiple characters since in Go single quotes represent a rune literal
+// https://github.com/golang/go/blob/65a54aef5bedbf8035a465d12ad54783fb81e957/src/strconv/quote.go#L360
 
 // unquoteSingle is the same as strconv.Unquote, but uses ' as a quote.
 func unquoteSingle(s string) (string, error) {
+	n := len(s)
+	if n < 2 {
+		return "", fmt.Errorf("%+q is not a quoted string", s)
+	}
+	quote := s[0]
+	if quote != s[n-1] {
+		return "", fmt.Errorf("%+q does not begin and end with a quote", s)
+	}
+	s = s[1 : len(s)-1]
+
+	if contains(s, '\n') {
+		return "", fmt.Errorf("%+q contains EOL", s)
+	}
+
+	// Is it trivial? Avoid allocation.
+	if !contains(s, '\\') && !contains(s, quote) {
```

Comment:
I think we can drop the `contains(s, '\\')` call, since it is already handled efficiently by `replaceEscapedMaybe` and saves us an iteration over the string.

Alternatively, maybe consider `if !strings.ContainsAny(s, "\\"+quote) {` which will reduce the test to one iteration.

Reply:
Again - this is a copy from stdlib.

`replaceEscapedMaybe` only handled the JS escape sequence and this is fast path for cases when the whole string has nothing to unquote.

Removing this shortcut would not affect the correctness, but it just follows the logic of https://github.com/bblfsh/javascript-driver/pull/64#discussion_r265019605